flake.nix

```diff
@@ -83,6 +88,8 @@
             self'.devShells.rust
           ];
           OM_INIT_REGISTRY = inputs.self + /crates/flakreate/registry;
+          NIX_FLAKE_SCHEMAS_BIN = lib.getExe (if pkgs.stdenv.isLinux then inputs'.nix.packages.nix-static else inputs'.nix.packages.default);
+          DEFAULT_FLAKE_SCHEMAS = "github:DeterminateSystems/flake-schemas";
```

Comment:
This should be pinned, exactly like nixci pins devour_flake. This is why CI is failing, because it is trying to fetch latest.

<img width="833" alt="image" src="https://github.com/user-attachments/assets/8636b561-e39c-4ee2-896b-c68f10d153f7">

Reply:
Step 1,

https://github.com/juspay/omnix/blob/9520311fb6b585526f2b4793b5cb956fcc1306de/flake.nix#L18-L19

Step 2,

https://github.com/juspay/omnix/blob/9520311fb6b585526f2b4793b5cb956fcc1306de/nix/rust.nix#L54

Reply:
Done in https://github.com/juspay/omnix/pull/166/commits/10d4bb9718923bce9b89b7343e7f2d6273ad8bf1

Reply:
I went ahead and pinned it by commit: https://github.com/juspay/omnix/pull/166/commits/37241cdb6229c204a8b98397039cef45333da17f